tests/rules/select-style.test.ts

```diff
@@ -56,67 +56,115 @@ ruleTester().run(path.parse(__filename).name, rule, {
+          constructor(private store: Store) {}
         }
       `,
+      {
+        options: [METHOD],
+        output: stripIndent`
+          import { Store } from '@ngrx/store'
+          @Component()
+          export class FixtureComponent {
+            foo$ = this.store.select(selector, selector2).pipe( filter(Boolean))
+            constructor(private store: Store) {}
+          }
+        `,
+      },
     ),
     fromFixture(
       stripIndent`
-        import { Store, select } from '@ngrx/store'
+        import { Store } from '@ngrx/store'
         @Component()
         export class FixtureComponent {
-          foo$;
-
-          constructor(private store: Store){
-            this.foo$ = store.pipe(select(selector));
-                                   ~~~~~~~~~~~~~~~~   [${methodSelectMessageId}]
+          foo$
+          constructor(private store: Store) {
+            this.foo$ = store.select(selector)
+                              ~~~~~~ [${operatorSelectMessageId}]
           }
         }
       `,
+      {
+        options: [OPERATOR],
+        output: stripIndent`
+          import { Store, select } from '@ngrx/store'
+          @Component()
+          export class FixtureComponent {
+            foo$
+            constructor(private store: Store) {
+              this.foo$ = store.pipe(select(selector))
+            }
+          }
+        `,
+      },
     ),
     fromFixture(
       stripIndent`
         import { Store } from '@ngrx/store'
         @Component()
         export class FixtureComponent {
-          foo$ = this.store.pipe(select(selector));
-                                 ~~~~~~~~~~~~~~~~   [${methodSelectMessageId}]
-
-          constructor(private store: Store){}
+          foo$ = this.store.pipe(select(selector), map(toItem)).pipe()
+                                 ~~~~~~ [${methodSelectMessageId}]
+          constructor(private store: Store) {}
         }
       `,
       {
         options: [METHOD],
+        output: stripIndent`
+          import { Store } from '@ngrx/store'
+          @Component()
+          export class FixtureComponent {
+            foo$ = this.store.select(selector).pipe( map(toItem)).pipe()
+            constructor(private store: Store) {}
+          }
+        `,
       },
     ),
     fromFixture(
       stripIndent`
+        import type {Creator} from '@ngrx/store'
         import { Store } from '@ngrx/store'
         @Component()
         export class FixtureComponent {
-          foo$ = this.store.select(selector);
-                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~       [${operatorSelectMessageId}]
-          constructor(private store: Store){}
+          foo$ = this.store.select(selector)
+                            ~~~~~~ [${operatorSelectMessageId}]
+          constructor(private store: Store) {}
         }
-
       `,
       {
         options: [OPERATOR],
+        output: stripIndent`
+          import { select } from '@ngrx/store';
```

Comment:
Would it be hard to add this to the existing import?

Reply:
If my implementation is correct, it should be fine now 😅